mParticle_AppboyTests/mParticle_AppboyTests.m

```diff
@@ -430,7 +430,8 @@ - (void)testlogPurchaseCommerceEvent {
                                        @"Total Amount" : @13.00,
                                        @"Total Product Amount" : @"13",
                                        @"Tax Amount" : @3,
-                                       @"Transaction Id" : @"foo-transaction-id"
```

Comment:
could we add example of all 3 of the keys to this test? Everything else looks great!